src/components/Submarine/Submarine.less

```diff
@@ -0,0 +1,214 @@
+
+			.lucid-Submarine-is-position-top& {
+				border-bottom: @border-standardBorder;
+				flex-direction: column-reverse;
+			}
+
+			.lucid-Submarine-is-position-bottom& {
+				border-top: @border-standardBorder;
+				flex-direction: column;
+			}
+
+			& > .lucid-Submarine-Bar-overlay {
+				position: absolute;
+				top: 0;
+				left: 0;
+				width: 100%;
+				height: 100%;
+				background-color: @color-backgroundColor;
+				opacity: 1;
+				z-index: @Submarine-Bar-zIndex;
+
+				.lucid-SplitHorizontal-is-expanded& {
+					opacity: 0;
+					pointer-events: none;
+				}
+			}
+
+			& > .lucid-Submarine-Bar-header {
+				background-color: @color-backgroundColor;
+				color: @color-textColor;
+				font: @size-L @fontFamily;
+				font-weight: bold;
+				display: flex;
+				flex-shrink: 0;
+				min-height: @Submarine-Bar-header-height;
+				.box-sizing();
+
+				.lucid-Submarine-is-position-top& {
+					border-top: @border-standardBorder;
+				}
+				.lucid-Submarine-is-position-bottom& {
+					border-bottom: @border-standardBorder;
+				}
+
+				& > .lucid-Submarine-Bar-Title {
+					margin: @size-XS auto @size-XS @size-standard;
+				}
+
+				& > .lucid-Submarine-expander {
+					.no-safari-flicker();
+					.box-sizing();
+					align-self: flex-start;
+					display: flex;
+					justify-content: center;
+					align-items: center;
+					flex-shrink: 0;
+					height: @size-expander-button;
+					width: @size-expander-button;
+					margin: @size-XS;
+					border: solid 1px @featured-color-default-borderColor;
+					border-radius: @size-borderRadius;
+					cursor: pointer;
+					z-index: @Submarine-Bar-zIndex + 1;
+					background-image: linear-gradient(@featured-color-default-gradientStartColor, @featured-color-default-gradientEndColor);
+
+					&:hover {
+						background-image: linear-gradient(@featured-color-default-gradientEndColor, @featured-color-default-gradientStartColor);
+					}
+
+					.lucid-ChevronIcon {
+						stroke: @color-darkGray;
+					}
+				}
+			}
+
+			& > .lucid-Submarine-Bar-content {
+				flex: 1 0;
+				padding: @size-XS;
+			}
+		}
+
+		& > .lucid-SplitHorizontal-Divider.lucid-Submarine-Divider {
+			position: relative;
+			display: flex;
+			flex-shrink: 0;
+			flex-direction: column;
+			background-color: @color-backgroundColor;
+			height: @Submarine-Divider-height;
+			z-index: @Submarine-Divider-zIndex;
+
+			.lucid-Submarine-is-position-top& {
+				transform: translateY(-@Submarine-Divider-height);
+				border-bottom: @border-standardBorder;
+			}
+
+			.lucid-Submarine-is-position-bottom& {
+				transform: translateY(@Submarine-Divider-height);
+				border-top: @border-standardBorder;
+			}
+
+			.lucid-SplitHorizontal-is-expanded& {
+				transform: translateY(0);
+			}
+
+			.lucid-Submarine-is-resize-disabled& {
+				display: none;
+			}
+
+			.lucid-Submarine-Divider-gripper {
+				margin: auto;
+			}
+
+			&:hover {
+				background-color: @color-mediumGray;
+
+				.lucid-Submarine-Divider-gripper {
```

Comment:
no, it needs to change it's fill color when the whole Divider line is hovered over, not just the gripper icon